chat_shell/chat_shell/tables/dingtalk.py

```
@@ -96,7 +96,15 @@ def _load_data_table_config(self) -> Optional[Dict[str, Any]]:
         Returns:
             Configuration dict or None if not set
         """
-        config_env = os.getenv("DATA_TABLE_CONFIG")
+        # Try to get from settings first (recommended way)
+        try:
+            from ..core.config import settings
+
+            config_env = settings.DATA_TABLE_CONFIG
+        except Exception:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)